datadog_checks_base/tests/test_tls.py

```diff
@@ -201,3 +201,47 @@ def test_client_key_expanded(self):
         with patch('ssl.SSLContext'), patch('os.path.expanduser') as mock_expand:
             check.get_tls_context()
             mock_expand.assert_called_with('~/foo')
+
+
+class TestTLSContextOverrides:
```

Comment:
I felt that it made more sense to have my `overrides` options testing in a new `TestTLSContextOverrides` instead of appending to existing ones, since this options affects multiple layers in the logic.